package.json

```diff
@@ -36,6 +36,7 @@
     "react-redux": "^4.4.1",
     "redux": "^3.3.1",
     "redux-thunk": "^2.0.1",
+    "sparql-client-2": "^0.6.0",
```

Comment:
Because I didn't realise Fabio had made a recommendation, I just picked any client that exposed a Promise interface. Shall move it over to this one (unless it's really really bad)